client/ayon_houdini/api/plugin.py

```diff
@@ -312,6 +326,48 @@ def apply_settings(self, project_settings):
         for key, value in settings.items():
             setattr(self, key, value)
 
+    def get_custom_staging_dir(self, product_type, product_name, instance_data=None):
```

Comment:
You know there is `get_staging_dir` on create plugin, which replaces almost everything you've implemented here?

Reply:
Does the note below help  answer that? https://github.com/ynput/ayon-houdini/pull/200/files#diff-02368e53ec1a73faaed14dfeee7bba367ff684790b1dd08cfe57a781422eeca5R338-R34

Reply:
NOTE: `instance._create_context` and `instance._creator.create_context` can be missing, but it is method on create plugin, so you have access to create plugin -> you have access to create context.